aws/data_source_aws_imagebuilder_image_test.go

```diff
@@ -0,0 +1,204 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/imagebuilder"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAwsImageBuilderImageDataSource_Arn_Aws(t *testing.T) {
```

Comment:
To help understand the naming rationale here, I'll use a generic example. Data sources tend to have varying argument requirements that change over time, so defining what a "basic" test might be over time could be challenging. 😄 

Say a data source currently requires an ARN argument. Later it is updated to make ARN optional and add an optional name argument. Which is "basic" now? Is there a "basic" test? It would feel odd to me to not update the original test name to differentiate what its trying to test and potentially hard to catch that during reviews. Rather than have this problem later on, seems like we can solve for it upfront by naming tests based on data source arguments. I think the only guaranteed "basic" case for a data source is when it can accept no arguments and it might be odd to name the test "_NoArguments" or something. Maybe it would not be odd though since its more explicit.

To be fair, we also have this problem for resources and "basic" tests being for "all required arguments", it is just much less likely to happen over time. I'll leave myself a note to update the acceptance testing documentation in some fashion for this topic, since its not defined well. Relatedly, at some point we need to make decisions on https://github.com/hashicorp/terraform-provider-aws/issues/13128, which may or may not be defined by the documentation update we can collaborate on. 😉 

If you feel strongly that this should be defined in a specific manner, let's discuss further in a new issue or in the change request for the documentation. There is also https://github.com/hashicorp/terraform-provider-aws/issues/8223 floating around out there, which wouldn't necessarily cover these naming situations completely, but could materially change the problem.